src/Android/Xamarin.Android/Samples/Network Analysis/FindServiceAreasForMultipleFacilities/readme.md

```diff
@@ -0,0 +1,38 @@
+# Find service areas for multiple facilities
+
+Find the service areas of several facilities from a feature service.
+
+![](FindServiceAreasForMultipleFacilities.jpg)
+
+## Use case
+
+A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be effectively accessed from particular hospitals, and with this optimize distribution of staff and resources.
```

Comment:
I think 
   "... areas of a city that can be effectively accessed from particular hospitals ..."
makes more sense as
   "... areas of a city that can be effectively served by each hospital ..."
since the direction of service (conceptually) is not going from hospitals to parts of the city but parts of the city accessing the hospital.